<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bdrSettingLoD2Dlg</class>
 <widget class="QDialog" name="bdrSettingLoD2Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Building Reconstruction LOD2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Config File   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ConfigureFilePathLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="ConfigureFilePathToolButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>高级配置</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>保存配置文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ground Height</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="GroundHeightUserRadioButton">
        <property name="text">
         <string>User Defined</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="GroundHeightContourRadioButton">
        <property name="text">
         <string>Contour Height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="GroundHeightLowestRadioButton">
        <property name="text">
         <string>Lowest Point</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="GroundHeightUserDoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outlineExtractionGroupBox">
     <property name="title">
      <string>Outline Extraction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="alphaDoubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="simplifyIntersectionDoubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>0.800000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>alpha-shape</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>执行简化平面最小面积</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="simplifyMinAreaDoubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> m2</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>简化求交阈值</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="outlineCDTCheckBox">
        <property name="text">
         <string>S-CDT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="regularizeCheckBox">
        <property name="text">
         <string>规则化</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="regularizeAngleDoubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> °</string>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rooflayerExtractionGroupBox">
     <property name="title">
      <string>Roof Layer Extraction</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="3">
       <widget class="QSpinBox" name="fpRoofLayerCompoMinPtsSpinBox">
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>层间最小距离</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>最小点数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="fpRoofLayerCompoDistDoubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="masterFootprintCheckBox">
        <property name="text">
         <string>建筑物足迹</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QGroupBox" name="footprintPolygonPartitionGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>水平拓扑优化</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="fpDataPtsRatioDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>数据项点数占比</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QSpinBox" name="fpSmoothSharpSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>数据项尺度因子占比</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>! 数据项尺度因子+平滑项尺度因子=1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>平滑项边界约束权值</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="fpDataRatioDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="roofTopologyGroupBox">
     <property name="title">
      <string>Roof Topology</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="roofStructureGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3"/>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="roofStructureRadioButton">
        <property name="text">
         <string>结构点</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="roofCDTRadioButton">
        <property name="text">
         <string>约束三角网标签选择</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="roofCDTGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="cdtDataRatioDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>数据项点数占比</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="cdtSmoothSharpSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>平滑项边界约束权值</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>数据项尺度因子占比</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>! 数据项尺度因子+平滑项尺度因子=1</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cdtHoleFillingCheckBox">
           <property name="text">
            <string>hole filling</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="cdtDataPtsRatioDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Vertical Buffer</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Horizontal Offset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="cdtHOffsetDoubleSpinBox">
           <property name="suffix">
            <string> m</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="singleStep">
            <double>0.200000000000000</double>
           </property>
           <property name="value">
            <double>0.200000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="cdtVerticalDoubleSpinBox">
           <property name="suffix">
            <string> m</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="singleStep">
            <double>0.200000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="cdtFitFootprintCheckBox">
           <property name="text">
            <string>Fit Footprint</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="cdtMaxIterCheckBox">
           <property name="text">
            <string>max iteration</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="cdtMaxIterSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lod3GroupBox">
     <property name="title">
      <string>LoD3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>bdrSettingLoD2Dlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>bdrSettingLoD2Dlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GroundHeightUserRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>GroundHeightUserDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roofCDTRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>roofCDTGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roofCDTRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>roofStructureGroupBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roofStructureRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>roofStructureGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roofStructureRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>roofCDTGroupBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cdtMaxIterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>cdtMaxIterSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regularizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>regularizeAngleDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
